src/SFA.DAS.ApprenticeFeedback/Application/Queries/GetApprentice/GetApprenticeQueryHandler.cs

```diff
@@ -33,6 +37,7 @@ public async Task<GetApprenticeResult> Handle(GetApprenticeQuery request, Cancel
                 Email = result.Email,
                 TermsOfUseAccepted = result.TermsOfUseAccepted,
                 ReacceptTermsOfUseRequired = result.ReacceptTermsOfUseRequired,
+                ApprenticePreferences = apprenticePreferences.ApprenticePreferences
```

Comment:
This *could* be an empty list, it shouldn't be, but if it was the deriving page that shows them dynamically would just be empty.

Would potentially be more resilient, what do you think?